app/views/static_pages/home.html.erb

```diff
@@ -1,174 +1,18 @@
-    <% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in Case. </br>
-    <% end %>
-</div>
-
-<div id= "linde" class = "dorms">
-Linde </br>
-There are <% dorm = Dorm.find_by_name("Linde") %>
-                        <%= dorm.unassigned_rooms_number %> empty rooms in Linde. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in Linde. </br>
-<% end %>
-
-</div>
-
-<div id= "sontag" class = "dorms">
-
-There are <% dorm = Dorm.find_by_name("Sontag") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in Sontag. </br>
-<% if current_user.student %>
-There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in Sontag. </br>
-<% end %>
-</div>
-
-<div id= "drinkward" class = "dorms">
-
-There are <% dorm = Dorm.find_by_name("Drinkward") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in Drinkward. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in Drinkward. </br>
-<% end %>
-</div>
-
-<div id= "atwood" class = "dorms">
-There are <% dorm = Dorm.find_by_name("Atwood") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in Atwood. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in Atwood. </br>
-<% end %>
-</div>
-
-<div id= "south" class = "dorms">
-There are <% dorm = Dorm.find_by_name("South") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in South. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in South. </br>
-<% end %>
-</div>
- 
-<div id= "north" class = "dorms">
-There are <% dorm = Dorm.find_by_name("North") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in North. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in North. </br>
-<% end %>
-</div>
-
-<div id= "east" class = "dorms">
-There are <% dorm = Dorm.find_by_name("East") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in East. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in East. </br>
-<% end %>
-</div>
-
-<div id = "west" class = "dorms">
-There are <% dorm = Dorm.find_by_name("West") %>
-                            <%= dorm.unassigned_rooms_number %> empty rooms in West. </br>
-<% if current_user.student %>
-    There are approximately <%= pullable_rooms_number(dorm) %> pullable rooms in West. </br>
-<% end %>
-</div>
-
-
 
+<% if current_user == nil %>
+    <%= render partial: "logged_out_home" %>
+<% elsif current_user.is_admin %>
+    <%= render "static_pages/admin_home" %>
```

Comment:
Admins who are students will need to be able to see the student homepage to pull their own room, I'd think?

Reply:
Never mind; see slack discussion